src/lib.cairo

```diff
@@ -38,6 +39,9 @@ mod components {
         }
         mod jpegs {
             mod farmer;
+            mod parrot;
+            mod swamp;
+            mod monkey;
```

Comment:
What about if another project is a different monkey? Maybe we should keep the project names.
Or be more precise on the animal type?

Reply:
Noted, might change in another PR